### PR TITLE
chore: Set jenkins pods resource limits

### PR DESCRIPTION
**Pull request by connoratrug**
- setting limit only sets requests equal to limit

### CODE REVIEW

**Review by christianhilbrands — CHANGES_REQUESTED** [review]
see comment above, could you write it like the example code given above?

**Comment by connoratrug** [comment]
> see comment above, could you write it like the example code given above?

why ? why not have i the same , you seem to no more about this that i do , please update the PR the way you think it should be.

File: Jenkinsfile [hunk 1]
```diff
@@ -73,7 +73,7 @@ pipeline {
                     sh "sleep 15s" // wait for deletion
                     sh "rancher apps install " +
                         "-n ${NAME} " +
-                        "p-tl227:molgenis-helm3-emx2 " +
+                        "c-l4svj:molgenis-helm3-emx2 " +
```
Comment (mswertz) [inline Jenkinsfile:76]: Has this changed again?

Comment (christianhilbrands) [reply Jenkinsfile:76]: yes

**Review by christianhilbrands — APPROVED** [review]
LGTM

**Pull request merged**